app/src/main/java/com/github/fribourgsdp/radio/GameActivity.kt

```diff
@@ -190,6 +207,23 @@ open class GameActivity : AppCompatActivity(), GameView {
         voiceChannel.joinChannel(voiceChannel.getToken(userId, gameUid.toString()), gameUid.toString(), "", userId)
     }
 
+    override fun onDone() {
+        // Run on main thread
+        runOnUiThread {
+            // When the timer is done it means the user didn't guess in time
+            // We display this in the same box as the sound so that is hides the guess input view
```

Comment:
small typo in documentation

Reply:
(on second documentation line)

Reply:
You're right I'm gonna correct it right away!